### PR TITLE
Load React asynchronously

### LINKED ISSUE #216: Delay downloading React
We shouldn't need React in the initial chunk, which really just needs to show the loading screen. By asynchronously importing React, we can start downloading the WASM a bit earlier and thus end the loading screen a bit earlier, as the WASM download is the bottleneck.

### PR DESCRIPTION
**Pull request by wooferzfg**
Fix #216 

Before:
before

After:
after

### CODE REVIEW

File: src/index.tsx [hunk 1]
```diff
@@ -34,12 +39,20 @@ Those are known to block WebAssembly.`);
...
+    if (!uiLoaded) {
+        ReactDOM.render(
+            <div className="initial-load">
+                <div className="fa fa-spinner fa-spin"></div>
+                <div className="initial-load-text">Loading...</div>
+            </div>,
+            document.getElementById("base"),
+        );
+    }
+}
```
Comment (CryZe) [inline src/index.tsx]: You don't need this function. You can just move this div into the HTML template (src/index.html).

Comment (wooferzfg) [reply src/index.tsx]: When I move it into the HTML template, the CSS files that we import in `index.tsx` don't get applied to it, so the loading screen looks like this:

chrome_J3GBhIiNLp

Comment (CryZe) [reply src/index.tsx]: You likely kept `className`. You need to rename it to `class`. It's `className` in jsx / tsx because `class` is a keyword.

Comment (wooferzfg) [reply src/index.tsx]: oh oops, i'm dumb lol

**Pull request merged**